api/proxy/config/app_config.go

```diff
@@ -51,21 +51,29 @@ func (c AppConfig) Check() error {
 	return nil
 }
 
-func ReadAppConfig(ctx *cli.Context) (AppConfig, error) {
+func ReadAppConfig(ctx *cli.Context, version string) (AppConfig, error) {
 	storeBuilderConfig, err := builder.ReadConfig(ctx)
 	if err != nil {
 		return AppConfig{}, fmt.Errorf("read proxy config: %w", err)
 	}
 
 	enabledServersCfg := enablement.ReadEnabledServersCfg(ctx)
+	restPublicInfo := rest.PubliclyExposedInfo{
+		Version:             version,
+		Network:             storeBuilderConfig.ClientConfigV2.EigenDANetwork.String(),
+		DirectoryAddress:    storeBuilderConfig.ClientConfigV2.EigenDADirectory,
+		CertVerifierAddress: storeBuilderConfig.ClientConfigV2.EigenDACertVerifierOrRouterAddress,
+		MaxBlobSizeBytes:    storeBuilderConfig.ClientConfigV2.MaxBlobSizeBytes,
```

Comment:
Does a client actually care about the `MaxBlobSizeBytes` or the `MaxSupportedPayloadSizeBytes` where:
```
MaxSupportedPayloadSizeBytes --codec_encode --> blob with length of MaxBlobSizeBytes
```

Reply:
Changed to `MaxPayloadSizeBytes` in [d2317d6](https://github.com/Layr-Labs/eigenda/pull/2219/commits/d2317d6ff35d53a03bb8da1811190f6391961611)